client/src/API.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+export default {
+    getPokemonName: function () {
+        const min = 4;
+        const max = 11;
+        return axios.get(`https://uzby.com/api.php?min=${min}&max=${max}`);
+        // return axios.get('/api/pokemonName');
+    },
+
+    createPokemon: function (payload) {
+        const config = {
+            method: 'post',
+            url: '/api/pokemon',
+            data: {
+                name: payload.name,
+                description: payload.desc,
+                image: payload.image,
+                type1: payload.type1,
+                type2: payload.type2,
+                moves: payload.moves
+            }
+        };
+        return axios(config);
+    }
+}
```

Comment:
add newlines at the end of each file so that git diffs will categorize new code there as additions and not changes (see [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file#:~:text=The%20%5C%20No%20newline%20at%20end,to%20take%20those%20in%20account) for more info)